pkg/controllers/dynakube/otelc/service/reconciler.go

```diff
@@ -15,13 +15,21 @@ import (
 
 const (
 	portNameZipkin              = "zipkin"
+	portZipkin                  = 9411
```

Comment:
idk why it was called portNameZipkin in first place, but IMO it shou;d be another way around:
```
zipkinPort
zipkinPortName
```

for example https://github.com/dynatrace/dynatrace-operator/blob/8cccb6ae2c523291e4ddd9d87d38dd3960cb231f/pkg/controllers/dynakube/extension/service.go#L72

and 

https://github.com/dynatrace/dynatrace-operator/blob/b94a1c2f61dcc085b36c60c88a21d92d128fdc66/pkg/controllers/dynakube/activegate/consts/consts.go#L13-L17